Userland/Libraries/LibGfx/TextAttributes.h

```diff
@@ -12,10 +13,18 @@
 namespace Gfx {
 
 struct TextAttributes {
+    enum class UnderlineStyle {
+        Solid,
+        Wavy
+    };
+
     Color color;
-    Optional<Color> background_color;
+    Optional<Color> background_color {};
```

Comment:
The braces are not needed here.

Reply:
There are some places that create `TextAttributes` without specifying (or need to) those Optionals. So I just empty-initialize them here to not do that at every single use site.
The only thing that is really needed is `color` anyways

Reply:
`Optional<T>`'s default ctor initialises the...ah, I see what you mean, yeah sure.